build.gradle

```diff
@@ -79,6 +79,13 @@ intellij {
 
 runIde {
     systemProperties.put("idea.log.debug.categories", "#com.leinardi.pycharm.mypy")
+    // Log verbose information when dynamic plugin unloading fails
+    systemProperties.put("ide.plugins.snapshot.on.unload.fail", "true")
+}
+
+// Causes error popup when building while sandbox IDE is open. Disable in development
+if (System.getenv('DEVELOP')) {
+    buildSearchableOptions.enabled = false
```

Comment:
Do you have any better ideas how to disable this conditionally in development only? (When running the buildPlugin task)

Right now I changed IDE run configuration to set the "DEVELOP" environment variable. Works well enough for me.

Reply:
Not really, I think env variables are a good option in this case.